packages/evolution-generator/src/scripts/generate_section_configs.py

```
@@ -71,17 +73,69 @@ def generate_section_configs(excel_file_path: str, section_config_output_folder:
+                    for n in (
+                        enable_conditional_name,
+                        completion_conditional_name,
+                    )
+                )
+
+                # Add generator comment
+                ts_section_code += add_generator_comment()
+
+                # Add imports
+                if needs_is_section_completed:
+                    ts_section_code += f"import {{ isSectionCompleted }} from 'evolution-common/lib/services/questionnaire/sections/navigationHelpers';\n"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)